examples/schelling_false.py

```diff
@@ -91,7 +88,6 @@ def main(args):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="parse args")
-    parser.add_argument('-n', '--num-samples', default=10, type=int)
```

Comment:
Removed this argument since we can get the mean over the EmpiricalDistribution directly, rather than generating samples and computing the mean.

Reply:
Beware that this might make the test invisible to our example test discovery kludge. Could you confirm that `schelling_false.py` is still tested, or muck with test_examples.py to ensure it is?

Reply:
I only ran this locally, and forgot about the `test_examples` kludge. Will check to see if that is running as expected.